src/assets/styles/main.scss

```diff
@@ -1,5 +1,5 @@
 @import 'normalize';
-@import 'colors';
+@import 'colours';
```

Comment:
I think US is better than GB here..
I used to change from favourite to favorite because US

Reply:
I did prefer ´colors´ but something weird was going on, ´_colors´ was conflicting with some other ´partial´ from Storybook.